Modules/+Drivers/ArduinoServo.m

```diff
@@ -0,0 +1,59 @@
+classdef ArduinoServo < Modules.Driver
+    %ARDUINOSERVO Connects with a hwserver Arduino and uses one pin to control a servo.
+    %
+    % Call with the 1) hostname of the host computer (singleton based on host), and 2) the (integer) pin.
+    
+    properties (Constant)
+        hwname = 'Arduino';
```

Comment:
Minor, but I believe other things that call hwserver use all lowercase?

Reply:
It's capitalized internally on hwserver, if I recall correctly.

Reply:
Is that the only one that's like that? If so, can we update to be consistent?